.github/workflows/publish.yml

```diff
@@ -44,7 +61,7 @@ jobs:
           # sadly, our build sometimes times out on network access
           # and running out of disk space: re-trying for 3 times
           for i in 1 2 3; do
-             if make V=1 LINUXKIT_PKG_TARGET=push pkgs; then
+             if make -e V=1 LINUXKIT_PKG_TARGET=push pkgs; then
```

Comment:
why `-e` is now necessary? Is it some of riscv specifics?

Reply:
This is just to pick additional variable settings from the ENV (see the section above that adds a few things like PKGS=xxx to the environment)

Reply:
Yeah, I was wondering about that as well.